### PR TITLE
Getting all the variables in app.py

### LINKED ISSUE #154: To get all variables
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/meilix

The idea is to get all the variables present in the index.html to app.py

### PR DESCRIPTION
**Pull request by meets2tarun**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes (provide a screenshot or link for test) 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

Fixes #154

### CODE REVIEW

**Comment by meets2tarun** [comment]
https://github.com/fossasia/meilix-generator/pull/152#pullrequestreview-128525743

**Comment by meets2tarun** [comment]
@niccokunzmann Is this that we need to do for getting all the variables. 
It needs improvement, some indentation error.

**Comment by niccokunzmann** [comment]
Yes, I agree.

**Comment by meets2tarun** [comment]
@niccokunzmann Please review once the changes.

Thanks

File: app.py [hunk 1]
```diff
@@ -36,7 +36,10 @@ def index():
         email = request.form['email']
         TRAVIS_TAG = request.form['TRAVIS_TAG']
         event_url = request.form['event_url']
-        GENERATOR_ = request.form['GENERATOR_']
+        variables = {}
+        for name, value in request.form.items():
+          if name.startswith("GENERATOR_"):
```
Comment (niccokunzmann) [inline app.py]: Usually, one uses four spaces in Python.

**Comment by niccokunzmann** [comment]
From my perspective, this is usable code. However, the indentation can be changed.

**Comment by niccokunzmann** [comment]
I think, if you solve #124 now, the passing of the variables to travis will be much easier than when you pass them to the script.

**Pull request merged**